Makefile

```diff
@@ -91,11 +91,13 @@ test: .develop
 
 .PHONY: vtest
 vtest: .develop
-	@pytest -s -v
+	@pytest -s -v -m "not dev_mode"
+	@python -X dev -m pytest -s -v -m dev_mode
```

Comment:
Can we try adding the 'not dev_mode' option by default, so a simple `pytest` will still run correctly?
https://stackoverflow.com/a/64729160

Reply:
Sure.